### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by morganestes**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/morganestes/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
```
Comment (morganestes) [inline ReadMe.md:2]: Well, thank you!

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (morganestes) [inline ReadMe.md:1]: !fix

Comment (morganestes) [inline ReadMe.md:1]: !fixed

**Pull request merged**